kornia/losses/focal.py

```diff
@@ -122,3 +122,93 @@ def forward(  # type: ignore
             input: torch.Tensor,
             target: torch.Tensor) -> torch.Tensor:
         return focal_loss(input, target, self.alpha, self.gamma, self.reduction, self.eps)
+
+
+def binary_focal_loss_with_logits(
+        inp: torch.Tensor,
+        target: torch.Tensor,
+        alpha: float = .25,
+        gamma: float = 2.0,
+        reduction: str = 'none',
+        eps: float = 1e-8) -> torch.Tensor:
+    r"""Function that computes Binary Focal loss.
+
+    """
```

Comment:
it's worth adding the documentation here too, including a very small example. Check e.g the functionalities from kornia.color

Reply:
Looking at the dice loss, I noticed that there is an "Examples" section at the module (and not at functional) so I added it there as well, just to be consistent.